canonical-web-and-design/assets.ubuntu.com
# Flask - Asset views

## Done
Add asset views to the flask application.

## QA
- `docker-compose up -d` to bring up Postgres and Swift
- `./run exec alembic upgrade head`
- `./run exec flask token create <name>`
- `./run`
- `sudo snap install upload-assets`
- `export UPLOAD_ASSETS_API_TOKEN=<token> `
- upload-assets -i -d localhost:8017 <asset_file>`
- Browse `localhost:8017?token=<token>` and make sure you can see the asset list
- Browse `localhost:8017/<file_path>` and make sure you can see the asset
- Browse `localhost:8017/<file_path>/info?token=<token>` and make sure you can see the asset info
